workflow/envs/mutational_context.yaml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - biopython =1.83
+  - pysam =0.22
+  - python >=3.7
```

Comment:
Rather pin the python version to something. Further, instead of 3 envs, just have one for all cases. Maybe call it pystats. The list of tools is not so large that we really need three.

Reply:
Joining the `altair`- and `mutational_context`-env into a single one makes sense. But do we really want to join siglasso as it contains R-packages.